projects/dotnet-micro-framework.md

```diff
@@ -1,15 +1,15 @@
 # .NET Micro Framework
 
-The [Microsoft® .NET Micro Framework](https://netmf.codeplex.com/) combines the reliability and efficiency of managed code with the premier development tools of Microsoft Visual Studio® to deliver exceptional productivity for developing embedded applications on small devices. 
+The [Microsoft® .NET Micro Framework](https://netmf.github/io/) is an open source platform that enables you to write managed code C# applications using Visual Studio for resource constrained embedded devices.
```

Comment:
@martinwoodward `https://netmf.github/io/` should be `https://netmf.github.io/` or the link won't work